src/features/defi/providers/osmosis/components/OsmosisManager/Deposit/components/Status.tsx

```diff
@@ -0,0 +1,207 @@
+import { CheckIcon, CloseIcon, ExternalLinkIcon } from '@chakra-ui/icons'
+import { Box, Button, Link, Stack } from '@chakra-ui/react'
+import { fromAccountId, osmosisAssetId } from '@shapeshiftoss/caip'
+import { Summary } from 'features/defi/components/Summary'
+import { TxStatus } from 'features/defi/components/TxStatus/TxStatus'
+import type {
+  DefiParams,
+  DefiQueryParams,
+} from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import { useCallback, useContext, useEffect, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+import { StatusTextEnum } from 'components/RouteSteps/RouteSteps'
+import { Row } from 'components/Row/Row'
+import { RawText, Text } from 'components/Text'
+import { useBrowserRouter } from 'hooks/useBrowserRouter/useBrowserRouter'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import {
+  selectAssetById,
+  selectFirstAccountIdByChainId,
+  selectMarketDataById,
+  selectTxById,
+} from 'state/slices/selectors'
+import { serializeTxIndex } from 'state/slices/txHistorySlice/utils'
+import { useAppSelector } from 'state/store'
+
+import { OsmosisDepositActionType } from '../DepositCommon'
+import { DepositContext } from '../DepositContext'
+
+export const Status = () => {
+  const translate = useTranslate()
+  const { state, dispatch: contextDispatch } = useContext(DepositContext)
+  const { query, history: browserHistory } = useBrowserRouter<DefiQueryParams, DefiParams>()
+  const { chainId } = query
+  const opportunity = state?.opportunity
+
+  const feeAsset = useAppSelector(state => selectAssetById(state, osmosisAssetId))
+  if (!feeAsset) throw new Error(`Fee asset not found for AssetId ${osmosisAssetId}`)
+  const feeAssetMarketData = useAppSelector(state =>
+    selectMarketDataById(state, osmosisAssetId ?? ''),
+  )
+
+  const underlyingAsset0 = useAppSelector(state =>
+    selectAssetById(state, opportunity?.underlyingAssetIds[0] ?? ''),
+  )
+  const underlyingAsset1 = useAppSelector(state =>
+    selectAssetById(state, opportunity?.underlyingAssetIds[1] ?? ''),
+  )
+  if (!underlyingAsset0)
+    throw new Error(`Asset not found for AssetId ${opportunity?.underlyingAssetIds[0]}`)
+  if (!underlyingAsset1)
+    throw new Error(`Asset not found for AssetId ${opportunity?.underlyingAssetIds[1]}`)
+
+  // user info
+  const accountId = useAppSelector(state => selectFirstAccountIdByChainId(state, chainId))
+  const userAddress = useMemo(() => accountId && fromAccountId(accountId).account, [accountId])
+
+  const serializedTxIndex = useMemo(() => {
+    if (!(state?.txid && userAddress && accountId)) return ''
+    return serializeTxIndex(accountId, state.txid, userAddress)
```

Comment:
This is probably where the `userAddress` need came from, taken from older implementations but this was a mistake we should tackle over all DeFi providers, in the context of EVM or Cosmos SDK chains where `fromAccountId(accountId).account` is deterministically the same address every time, we shouldn't need addresses